query-engine/connector-test-kit-rs/query-test-macros/src/connector_test.rs

```diff
@@ -91,10 +91,11 @@ pub fn connector_test_impl(attr: TokenStream, input: TokenStream) -> TokenStream
                 query_tests_setup::run_with_tokio(async move {
                     tracing::debug!("Used datamodel:\n {}", datamodel.yellow());
 
+                    query_tests_setup::setup_project(&datamodel).await.unwrap();
```

Comment:
Any particular reason for that change?

Reply:
The binary runner needs the database to be setup and migrated before we can run the `binary_runner::setup` otherwise it tries to connect to the database which doesn't exist yet.